src/components/UserBackgroundImage.js

```diff
@@ -42,11 +58,31 @@ const UserBackgroundImage = ({ user }) => {
     backgroundImage: { timestamp: backgroundImageTimestamp, imageURL },
     id: userId,
   } = user
+  const [backgroundImages, setBackgroundImages] = useState([
+    { imageURL, preloaded: true },
+  ])
+
+  const fireOnLoad = () => {
+    setBackgroundImages((prevState) => [
+      prevState[prevState.length - 2],
+      { ...prevState[prevState.length - 1], preloaded: true },
+    ])
+  }
   useEffect(() => {
     // Show a new background image every day.
-
     async function updateBackgroundAndCachePage() {
-      await SetBackgroundDailyImageMutation(userId)
+      const {
+        setUserBkgDailyImage: {
+          user: {
+            // eslint-disable-next-line no-shadow
+            backgroundImage: { imageURL },
```

Comment:
Reassign to a new variable name instead of disabling no-shadow.

Reply:
I get what you're saying—but these variables are distinct values serving different purposes, so it's worth calling that out.  It's easy to imagine logging to debug this and accidentally misunderstanding whether you're looking at state or a mutation return value. You can capture your comment in the variable name, like `updatedImageURL`.